packages/widget-v2/src/pages/SwapExecutionPage/SwapExecutionPageRouteDetailedRow.tsx

```diff
@@ -226,7 +226,6 @@ const StyledLoadingContainer = styled(Row) <{
         return `border: ${borderSize}px solid ${theme.success.text}`;
       case "failed":
         return `border: ${borderSize}px solid ${theme.error.text}`;
-      case "broadcasted":
```

Comment:
redundant, falls back to default case in every other case no need to be explicit